.github/workflows/test-build.yaml

```diff
@@ -38,16 +38,29 @@ jobs:
       - name: Build
         run: make manager
 
-      - name: Login to GitHub Packages Docker Registry
+      - name: Docker meta
+        id: meta
+        uses: docker/metadata-action@v3
+        with:
+          # list of Docker images to use as base name for tags
+          images: |
+            ghcr.io/skyscanner/applicationset-progressive-sync
```

Comment:
We're now pushing to Github packages when opening a PR. This allow us to pull a specific docker image for e2e testing before merging it into main.